09-gis.Rmd

```diff
@@ -24,7 +24,7 @@ Other 'command-lines' include terminals for interacting with Linux, Mac and Wind
 In fact, most early computers and GISs such had GRASS relied exclusively on CLIs, before the widespread uptake of computer mice and high resolution screens in the 1990s.
 ]
 In dedicated GIS packages, by contrast, the emphasis tends to be on the graphical user interface (GUI):
-you *can* interact with QGIS, SAGA and gvSIG with dockable CLIs, but 'pointing and clicking' is the norm.
+you *can* interact with GRASS (the longest-standing GIS CLI), QGIS, SAGA and gvSIG with dockable CLIs, but 'pointing and clicking' is the norm.
```

Comment:
mmh, as usual I would be a bit more careful, it is only the longest-standing GIS CLI of the here listed GIS

Reply:
No, it's the longest standing GIS CLI, by which I mean the longest in continuous development. That's well documented. @veroandreo may be able to provide more info on this but I'm confident this is correct. If you know of any older GIS CLI's still standing, let us know!

Reply:
I would just rephrase to something like `the longest standing GIS CLI of the here listed GIS)`

Reply:
Here some notes from the  founding software architect of GRASS: https://grass.osgeo.org/home/history/van-warren-notes/ and a list of releases (scroll to the very bottom): https://grass.osgeo.org/home/history/releases/
AFAIU, first version of GRASS precursor was delivered in 1982.

Reply:
That's pretty long standing and supports the statement it's the 'longest standing' GIS. Many thanks for the info.

Reply:
just because you wish it were the 'longest standing' CLI GIS does not make it the longest standing -:). In fact, the first official GRASS release was in 1984. ArcInfo was released in 1982 for the first time (as a CLI, in fact). And as I said before, the first GIS ever (CGIS (early 60s), Symap (70s)) were probably also based on the cli. But let's not drown in a puddle, I will simply adjust the statement.

Reply:
Yes, to set it right, I was always referring to FOSS (free and open source) GIS. So, if not too clumpsy in the flow of the text it could be re-phrased to `the longest-standing free and open source GIS programs...` In any case, the latest update looks good to me too. Thanks much for this great effort of yours!